### PR TITLE
Transactions and Options

### LINKED ISSUE #3: Issue/2  PDF Downloadable

### PR DESCRIPTION
**Pull request by ishgary**
### Resolve: #3 

**Welcome Page**
image

**Create Transaction**
image

**Transaction Listing**
image

### CODE REVIEW

File: app/Http/Controllers/transactionController.php [hunk 1]
```diff
@@ -83,4 +84,15 @@ public function destroy($id)
         $data->delete();
         return redirect()->route('listTrans');
     }
+    public function receipt ($id)
+    {
+        $data = tran::find($id);
+        $trans_data = [
```
Comment (Gauravbhatt19) [inline app/Http/Controllers/transactionController.php:90]:
```suggestion
        $transaction = [
```
Use descriptive names & also only use camelCase for identifiers.

Comment (Gauravbhatt19) [reply app/Http/Controllers/transactionController.php:90]: Similar changes throughout.

File: app/Models/tran.php [hunk 2]
```diff
@@ -19,9 +20,31 @@ class tran extends Model
...
+            $zeros = 0;
+            // if($last_record->date == $model->date)
+            // {
+            //     $prev_integer = (int)substr($last_record->invoice, -2);
+            //     $entry_number = $prev_integer+1;
+            //     if($entry_number>9)
+            //     {
+            //         $zeros = "";
+            //     }
+            // }
```
Comment (Gauravbhatt19) [inline app/Models/tran.php:44]:
```suggestion
```
Remove commented unnecessary code.

**Review by Gauravbhatt19 — APPROVED** [review]
Minor suggestions, rest looks good to merge 👍🏻

File: app/Models/tran.php [hunk 3]
```diff
@@ -19,9 +20,31 @@ class tran extends Model
...
         
     ];
 
+    public static function boot()
+    {
+        parent::boot();
+        static::creating(function($model){
+            $last_record = DB::table('trans')->latest()->first();
+            $entry_number = 0;
+            $zeros = 0;
```
Comment (AmanKala) [inline app/Models/tran.php:35]: I don't think that these two variables are useful as they are simply adding a couple of zeros at the end of invoice number, so you can remove them.

**Review by AmanKala — APPROVED** [review]
Just a minor suggestion, rest looks fine to me.

**Pull request merged**